pom.xml

```diff
@@ -642,6 +679,26 @@ under the License.
               <fail>true</fail>
             </configuration>
           </execution>
+          <execution>
+            <id>enforce-bytecode-version</id>
```

Comment:
```suggestion
            <id>enforce-build-jdk-version</id>
```

Reply:
This execution overrides the one defined in the parent pom, with the difference that it adds the `ignoredScopes` configuration bit.  So it cannot be renamed...

Reply:
Ah, wasn't aware of that. Makes sense then!